### DISCUSSION TITLE
The update on Columns outside SetupTables Column Range also marks the record as changed?

### DISCUSSION THREAD

**Discussion by ricardoamm** [Mimetis/Dotmim.Sync#826] [Q&A]
Hello,

I'm facing the following situation, the update on Columns outside SetupTables Column Range seems to mark the record as changed.
Could you clarify, please?

Regards

**Comment by Mimetis** [comment]
Yes, any columns will trigger the update trigger (or change tracking if you are using sql CHANGE_TRACKING)
